wcfsetup/install/files/lib/system/user/notification/event/TReactionUserNotificationEvent.class.php

```diff
@@ -0,0 +1,76 @@
+<?php
+namespace wcf\system\user\notification\event;
+use wcf\data\user\UserProfile;
+use wcf\system\database\util\PreparedStatementConditionBuilder;
+use wcf\system\user\notification\object\IUserNotificationObject;
+use wcf\system\WCF;
+
+/**
+ * This trait can be used in likeable user notification events to determine the reactionType counts
+ * for a specific user notification object. 
+ * 
+ * @TODO backport to WSC 3.1
```

Comment:
TODO

Reply:
I'm not sure if we should backport the class with the upcoming update. I think it's way better if we backport the class just before the 3.2 beta, if there are any changes (or a developer should come up with the idea of implementing the whole thing productively).
What do you think @dtdesign?